model/src/main/proto/oppia_logger.proto

```diff
@@ -118,6 +277,26 @@ message EventLog {
     // The priority of events whose logs can be removed from the storage if the size exceeds a certain limit
     OPTIONAL = 2;
   }
+
+  // Structure of the generic data.
+  message GenericData {
+    // The learner id of the current user. This id is profile-specific.
+    string learner_id = 1;
+    // The device id of the current device. This id is common across all profiles.
+    string device_id = 2;
+  }
+
+  // Structure of the exploration data.
+  message ExplorationData {
```

Comment:
Is suffixing these with 'Data' conventional? It might be clearer just to call this 'ExplorationDetails' or 'ExplorationContext' (though that might conflate with the event contexts above).

I am rethinking some of these names a bit since earlier naming choice deficiencies are becoming more noticeable now that a lot of new events are being added.

Reply:
No, its not conventional. I used because it was mentioned in the TDD. Changed to `ExplorationDetailsContext`.